dde-desktop/view/backgroundmanager.cpp

```diff
@@ -433,11 +433,13 @@ void BackgroundManager::onResetBackgroundImage()
         qDebug() << sp->name() << "background path" << userPath << "truesize" << trueSize << "devicePixelRatio"
                  << bw->devicePixelRatioF() << pix << "widget" << bw.get();
         pix.setDevicePixelRatio(bw->devicePixelRatioF());
+        //更新壁纸
```

Comment:
Update in cycle? This is definitely not a good idea.
In addition，bw->setPixmap(pix) has actually updated the picture.